Makefile

```diff
@@ -93,6 +93,14 @@ integration-test: ## Run the integration tests
 	$(MAKE) start-db
 	${DOCKER_COMPOSE} run --rm app npm run test:integration; exit=$$?; ${DOCKER_COMPOSE} down; exit $$exit
 
+api-validate: ## Run the API analysis
+	$(MAKE) start wait-healthy
+	docker run --rm --network host hydrofoil/hydra-analyser:0.2.0 http://localhost:8080/; exit=$$?; ${DOCKER_COMPOSE} down; exit $$exit
+
+api-test: ## Run the API tests
+	$(MAKE) start wait-healthy
+	docker run --rm --init --network host --mount "type=bind,source=$(CURDIR)/test/hypertest/,destination=/tests" hydrofoil/hypertest:_0.4.1 --baseUri http://localhost:8080/; exit=$$?; $(DOCKER_COMPOSE) logs; ${DOCKER_COMPOSE} down; exit $$exit
```

Comment:
`exit=$$?; $(DOCKER_COMPOSE) logs; ${DOCKER_COMPOSE} down; exit $$exit` seems error prone to duplicate and maintain. Can it be sticked into another Makefile variable like `DOCKER_COMPOSE`?

Reply:
Make sense. Already exists, so will extract the change.

Reply:
https://github.com/libero/article-store/pull/274

Will need a small change to add the logs back in.